spec/inputs/jms.yml

```diff
@@ -0,0 +1,18 @@
+---
+# HornetQ Broker
+hornetq:
+  # Connect to a local HornetQ Broker using JNDI
+  :jndi_name: /ConnectionFactory
+  :jndi_context:
+    java.naming.factory.initial: org.jnp.interfaces.NamingContextFactory
+    java.naming.provider.url: jnp://localhost:1099
+    java.naming.factory.url.pkgs: org.jboss.naming:org.jnp.interfaces
+    java.naming.security.principal: guest
+    java.naming.security.credentials: guest
+  :require_jars:
```

Comment:
we should move the jar dependency handling to the gemspec like https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/master/logstash-output-elasticsearch.gemspec#L23

this way the jars are pulled from maven automatically at install time. it will generate a .rb file with all the requires, so you need to also add `require 'logstash-input-jms_jars'` in line 5 of lib/logstash/inputs/jms.rb